src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -197,64 +188,56 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
       rows => rows.length > 0)
 
     // Post new assets
-    spark
+    val somedf = spark
```

Comment:
`someDf` but I see this is only for debugging ;)

Reply:
Yep!